src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -20,15 +20,19 @@ public class CompilerDataLogger
         private const string CommandLineEventName = "CommandLineInformation";
         private const string AssemblyReferencesEventName = "AssemblyReferencesInformation";
 
-        private readonly int ChunkSize;
-        private readonly string sha256;
+        private string Sha256 => context?.Hashes?.Sha256;
```

Comment:
>context [](http://example.com/codeflow?start=33&length=7)

the code previously seemed to return string.empty in the null case, is that still required or is it ok to drop it?

Reply:
when we send a null to AppInsights, it won't render the key value pair. If we try to print it in console, a null is showed as an empty string.

so, in this case, we don't need to create a default to empty